php-frontend/src/main/java/org/sonar/php/tree/visitors/FrameworkDetectionVisitor.java

```diff
@@ -32,6 +32,8 @@ public void visitUseClause(UseClauseTree tree) {
       this.framework = SymbolTable.Framework.DRUPAL;
     } else if (tree.namespaceName().qualifiedName().startsWith("Yii")) {
       this.framework = SymbolTable.Framework.YII;
+    } else if (tree.namespaceName().qualifiedName().startsWith("WP_CLI") || tree.namespaceName().qualifiedName().startsWith("WP\\")) {
```

Comment:
I think it's time to assign `tree.namespaceName().qualifiedName()` to a local var and validate it instead of retrieving the FQN every condition.